ci/update_securityhub_suppressions/suppressions.yml

```diff
@@ -210,7 +210,7 @@ Suppressions:
   "ebs-in-backup-plan-conformance-pack-ikljqduwd":
     - resources:
         - arn:aws:ec2:eu-west-2:776473272850:volume/vol-072201eb4a2eaf7e8
-        - arn:aws:ec2:eu-west-2:776473272850:volume/vol-09cd56b781f85cef6
+        - arn:aws:ec2:eu-west-2:776473272850:volume/vol-04812795b57f856f1
```

Comment:
Why did our EBS volume change? Was something else reconfigured in the EC2 instance?